src/connectionmanager.cpp

```diff
@@ -187,8 +187,6 @@ void ConnectionManager::implInitialized(bool status, bool a_connected,
                  << "connected:" << a_connected
                  << "connectionDate:" << connectionDate.toString();
 
-  Q_ASSERT(m_state == ConnectionManager::StateInitializing);
```

Comment:
Why was this behavior changed?

Reply:
When the daemon crashes, the controller attempts to re-initialize it, which will trigger the asynchronous emission of another `ControllerImpl::initialized()` signal from any state, therefore this assert is no longer valid.

For example, if the daemon crashed while in `ConnectionManager::StateOn` we will wind up here when re-starting the daemon.